metaflow/runtime.py

```diff
@@ -893,17 +921,22 @@ def terminate(self):
         # Return early if the task is cloned since we don't want to
         # perform any log collection.
         if not self.task.is_cloned:
-            self.task.save_logs('stdout', self._stdout.get_bytes())
-            self.task.save_logs('stderr', self._stderr.get_bytes())
+            self.task.save_logs(self._stdout.get_bytes(),
```

Comment:
I didn't quite understand this comment. Can you elaborate?

Reply:
I think I was trying to say something that this may be too soon if there is more log coming with the emit_log and task.log but looking at the code I see I am mistaken and that all log should be already there in the stdout/stderr.